source/adminguide/api.rst

```diff
@@ -87,4 +87,93 @@ is also supported.) For metadata type, use one of the following:
 
 -  instance-id. The instance name of the VM
 
+User Data and Meta Data via Config Drive
+----------------------------------------
 
+The user-data service on a Shared or L2 Network can be provided through the
+Virtual Router or through an attached iso called the Config drive.
```

Comment:
may be this ^^^ you can keep with heading 'User Data and Meta Data', and sub-heading 'via Virtual Router' and 'via Config Drive'

Reply:
Done

Reply:
L2 networks and Virtual Router???